src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -30,11 +30,17 @@ def bugzilla_url(email_list = '', desc = '')
     )
   end
 
-  def user_icon(user, size = 20, css_class = nil, alt = nil)
-    user = User.find_by_login!(user) unless user.is_a? User
-    alt ||= user.realname
+  def user_icon(user, opt = {})
```

Comment:
This method is used once. How about you drop this too? :-)

Reply:
I count 2 usages - once for the large icon on the user's home and once for small icons in a partial

Reply:
Oh well true, still rather overkill I would say...